app/services/operations/create_application.rb

```diff
@@ -13,6 +13,8 @@ def initialize(payload:)
     def call
       app = CrimeApplication.create!(application: payload)
 
+      SupersedeApplication.new(application_id: parent_id).call if parent_id
```

Comment:
I'm wondering if we want to wrap create and supersede in a transaction in case supersede! fails?

Reply:
Hmm, yeah, I think that might be a good idea because any error on the Superseded would bubble up and produce an API exception which on Apply will look like the submission failed, however the application may have been persisted.
Will make the change.